sprocs/ip_to_mode.sql

```diff
@@ -44,7 +49,48 @@ BEGIN
         AND (date BETWEEN r.access_start AND r.access_end);
 
     IF FOUND THEN
-        mode := 'Exam';
+        -- We have a checked-in pt_reservation. Check whether we are in a
+        -- testing center location that requires network filtering.
```

Comment:
The use of "we are in..." throughout the following few comments was a little confusing on the first read through since it's used to mean two different things. I'd recommend disambiguating by saying either "the reservation is in a testing center location..." or "the user is physically in the location...".

Reply:
Excellent point. The two uses of "we are in" were somehow clear in my head while writing, but they are completely unclear while reading! Fixed in 8ba21d5649f19cf40425509c18bb29f85c5d6ff2